src/applications/vaos/newAppointmentFlow.js

```diff
@@ -149,6 +156,16 @@ export default {
     next: 'vaFacility',
     previous: 'typeOfCare',
   },
+  typeOfEyeCare: {
+    url: '/new-appointment/choose-eye-care',
+    next(state) {
+      if (getFormData(state).typeOfEyeCareId === '407') {
```

Comment:
Another way of doing this might be to call the `getTypeOfCare` function and check that the result does not have a `ccId`. I think that's a more accurate reflect of what you're wanting to do here, which is to make sure the non-CC type of eye care goes to the va page.